pkg/detectors/shortcut/shortcut.go

```diff
@@ -48,23 +48,33 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.app.shortcut.com/api/v3/projects", nil)
```

Comment:
not every key will have a project, but every key will at a minimum have a member